nginx.conf

```diff
@@ -0,0 +1,33 @@
+upstream observation_portal {
```

Comment:
I kinda want this file to be in a separate deploy directory, to keep it a bit more organized.

Reply:
okay, should the docker-compose go in the deploy directory too?

Reply:
Yeah that sounds good to me